packages/usa-banner/src/test/template.html

```diff
@@ -9,38 +9,43 @@
           <p class="usa-banner__header-text">An official website of the United States government</p>
           <p class="usa-banner__header-action" aria-hidden="true">Here’s how you know</p>
         </div>
-        <button class="usa-accordion__button usa-banner__button"
-          aria-expanded="false"
-          aria-controls="gov-banner"
-        >
-          <span class="usa-banner__button-text">Here's how you know</span>
+        <button class="usa-accordion__button usa-banner__button" aria-expanded="false"
+          aria-controls="gov-banner-default">
+          <span class="usa-banner__button-text">Here’s how you know</span>
         </button>
       </div>
     </header>
-    <div class="usa-banner__content usa-accordion__content" id="gov-banner">
+    <div class="usa-banner__content usa-accordion__content" id="gov-banner-default" hidden="">
```

Comment:
Curious - why was hidden added here?

Reply:
Me too, not sure where this came from. It shouldn't be there and removed as part of b50e5a6.